### PR TITLE
style fix in OpenMPI easyconfigs

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/o/OpenMPI/OpenMPI-1.6.3-iccifort-2011.13.367.eb [hunk 1]
```diff
@@ -20,14 +20,11 @@ dependencies = [('hwloc', '1.6')]
...
 sanity_check_paths = {
     'files': ["bin/%s" % binfile for binfile in ["ompi_info", "opal_wrapper", "orterun"]] +
-             ["lib/lib%s.%s" % (libfile, SHLIB_EXT) for libfile in ["mca_common_sm", "mpi_cxx",
-                                                                         "mpi_f77", "mpi_f90",
-                                                                         "mpi", "ompitrace", "open-pal",
-                                                                         "otfaux", "otf",
-                                                                         "open-rte", "vt", "vt-hyb",
-                                                                         "vt-mpi", "vt-mpi-unify"]] +
+             ["lib/lib%s.%s" % (libfile, SHLIB_EXT) for libfile in libs],
              ["include/%s.h" % x for x in ["mpi-ext", "mpif-common", "mpif-config", "mpif",
```
Comment (wpoely86) [inline easybuild/easyconfigs/o/OpenMPI/OpenMPI-1.6.3-iccifort-2011.13.367.eb]: and this one?

**Comment by wpoely86** [comment]
looks fine?

File: easybuild/easyconfigs/o/OpenMPI/OpenMPI-1.6.3-iccifort-2011.13.367.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/o/OpenMPI/OpenMPI-1.6.3-iccifort-2011.13.367.eb]: the main issue was that the list of names was way to the right, this line doesn't suffer from that

**Comment by fgeorgatos** [comment]
visual review... LGTM

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 11 out of 13 (8 easyconfigs in this PR)
Linux RHEL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d5fc602af7d8aa4e5930 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
Linux RHEL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/35cb93f69f7265c2f9fa for a full test report.

**Pull request merged**